src/components/DonutChart/DonutChart.js

```diff
@@ -2,27 +2,36 @@ import React, { useRef, useEffect } from "react";
 import * as d3 from "d3";
 
 const DonutChart = ({ chartData }) => {
```

Comment:
See my comment below about `chartData` and splitting out into the three required values.